### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by janckerchen** [EA31337/EA31337-classes#27]

**Comment by kenorb** [comment]
Yes, there are some plans to continue the development, but limited time is currently used at other projects. I hope in further commits next month. Thanks for the interest.

**Comment by kenorb** [comment]
I've pushed +10 commits, further work is planned. If you've got any specific issues with the code or specific class, please raise a new ticket.

**Comment by kenorb** [comment]
Btw. New EA version has been released: https://github.com/EA31337/EA31337/releases

**Comment by kenorb** [comment]
@janckerchen I just noticed that you're the author of the GH-22. Sorry for not noticing it before:)

It took me a long while to get around your PR, and to merge it (as part of GH-50), as I wasn't sure how to implement it in the practical way, and I wasn't ready for it yet. I've renamed your class to `IndicatorData` to not conflict with the existing `Indicator` (GH-49), as I've manage to implement some similar functionality as well, and I want to compare both solutions. I'll be testing both `Indicator` and `IndicatorData` in practical sense, and it's going to be evolving slowly. I've checked and your class still works as expected.

Next, I'm planning to implement Strategy classes which are going to implement indicator data classes and I'll be finding out what's the best way to use these classes practically, what they'll require to work, and checking out the performance when I'll have dozens of indicators running at the same time.
